pavlolucik/docker/03nginxWithCustomStartPage/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+LABEL maintainer="Pavlo Lucik pavel.lucik@gmail.com"
+RUN apt-get update
```

Comment:
apt-get update && apt-get install - don't split it to two step as you'll use obsoleted package database in this case.